### ISSUE TITLE
Scraping from community-operators repo?

### ISSUE THREAD

**Issue by dmesser** [artifacthub/hub#811]
Hi,

I just noticed that all the Operators from OperatorHub.io seem to be scraped from https://github.com/operator-framework/community-operators and put on the hub, for example: https://artifacthub.io/packages/olm/community-operators/hazelcast-enterprise-operator

I am one of the maintainers behind this repo and just noticed this. Wondering if you were discussing this with anyone from the community operator registry project/OperatorHub.io?

/Daniel

**Comment by mattfarina** [comment]
@dmesser That's a great question.

While the source of the community operators repository is Apache 2 open source so Artifact Hub and others could use it, we wanted to be respectful and ask. I came to a meeting and asked about it. No one saw any issue with it. I went on to ask [a follow-up question on a mailing list](https://groups.google.com/g/operator-framework-olm-dev/c/3XccPL779JI) to dig in further on Artifact Hubs handling. At the time of the post no one said anything about using the metadata. I did a quick search of meeting videos to try and find a recording but was unable to find them.

I'd asked around informally prior to this and no issues were raised.
